wear/src/main/java/io/homeassistant/companion/android/sensors/SensorReceiver.kt

```diff
@@ -49,6 +50,10 @@ class SensorReceiver : SensorReceiverBase() {
             StepsSensorManager(),
             TheaterModeSensorManager()
         )
+        val MANAGERS = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R)
```

Comment:
Any specific reason you're doing it like this? The main app also has SDK limited sensor managers but still uses one list.

Reply:
Lint was failing so it was either the current implementation or we skipped the lint check here. I decided against skipping the lint check but in reality we can ignore it as the code will not be reached.

Reply:
Missed that one! Can't that be solved by using `@RequiresApi(Build.VERSION_CODES.R)`, like for various other sensors?

> but in reality we can ignore it

We can't as it is a requirement to merge PRs 🙃

Reply:
> Missed that one! Can't that be solved by using `@RequiresApi(Build.VERSION_CODES.R)`, like for various other sensors?

its already there in `HealthServicesSensorManager` but it wasn't good enough. For some reason it had a different complaint here than it did for DND sensor in the common folder.



> We can't as it is a requirement to merge PRs 🙃

lol I meant to suppress lint like one of Android Studio recommendations lol